spec/models/employment_tribunal_decision_spec.rb

```diff
@@ -0,0 +1,185 @@
+          "type" => "exact",
+        }
+      ],
+      "redirects" => [],
+      "update_type" => "major",
+    }
+  end
+
+  let(:non_employment_tribunal_decision_content_item) {
+    {
+      "content_id" => SecureRandom.uuid,
+      "base_path" => "/other-documents/not-an-employment-tribunal-decision",
+      "format" => "specialist_document",
+      "details" => {
+        "metadata" => {
+          "document_type" => "not_an_employment_tribunal_decision",
+        },
+      },
+    }
+  }
+
+  let(:employment_tribunal_decision_org_content_item) {
+    {
+      "base_path" => "/courts-tribunals/employment-tribunal",
+      "content_id" => "8bb37087-a5a7-4493-8afe-900b36ebc927",
+      "title" => "Employment Tribunal",
+      "format" => "placeholder_organisation",
+      "need_ids" => [],
+      "locale" => "en",
+      "updated_at" => "2015-08-20T10:26:56.082Z",
+      "public_updated_at" => "2015-04-15T10:04:28.000+00:00",
+      "phase" => "live",
+      "analytics_identifier" => "CO1133",
+      "links" => {
+        "available_translations" => [
+          {
+            "content_id" => "1b5e08c8-ddde-4637-9375-f79e085ba6d5",
+            "title" => "Employment tribunal decisions",
+            "base_path" => "/employment-tribunal-decisions",
+            "description" => nil,
+            "api_url" => "https://www-origin.integration.publishing.service.gov.uk/api/content/employment-tribunal-decisions",
+            "web_url" => "https://www-origin.integration.publishing.service.gov.uk/employment-tribunal-decisions",
+            "locale" => "en"
+          }
+        ]
+      },
+      "description" => nil,
+      "details" => {}
+    }
+  }
+
+  let(:indexable_attributes) {
+    {
+      "title" => "Example Employment Tribunal Decision 0",
+      "description" => "This is the summary of example Employment Tribunal Decision 0",
+      "link" => "/employment-tribunal-decisions/example-employment-tribunal-decision-0",
+      "indexable_content" => "## Header" + ("\r\n\r\nThis is the long body of an example Employment Tribunal Decision" * 10),
+      "public_timestamp" => "2015-11-16T11:53:30+00:00",
+      "tribunal_decision_categories" => ["age-discrimination"],
+      "tribunal_decision_country" => "england-and-wales",
+      "tribunal_decision_decision_date" => "2015-07-30",
+      "organisations" => ["employment-tribunal"],
+    }
+  }
+
+  before do
+    @fields = [
+      :base_path,
+      :content_id,
+    ]
+
+    @employment_tribunal_decisions = []
+
+    10.times do |n|
+      @employment_tribunal_decisions << employment_tribunal_decision_content_item(n)
+    end
+
+    publishing_api_has_fields_for_format('specialist_document', @employment_tribunal_decisions, @fields)
+
+    @employment_tribunal_decisions.each do |decision|
+      publishing_api_has_item(decision)
+    end
+
+    Timecop.freeze(Time.parse("2015-12-18 10:12:26 UTC"))
+  end
+
+  context "#all" do
```

Comment:
It's better to use `describe` for blocks describing the behaviour of a method, then `context` to define the context for a particular set of behaviours for that method/object.

http://betterspecs.org/#describe